src/main/kotlin/lotto/LottoApplication.kt

```diff
@@ -13,40 +13,38 @@ fun main() {
     val outputView = OutputView()
 
     // 구입금액 입력받기
-    val price = inputView.printMsgAndReadValue(Const.OutputMsg.GET_PRICE_MSG)
+    val price = inputView.getPrice()
 
     // 구입금액 validation
     val lottoPurchase = LottoPurchase()
-    val lottoPrice = lottoPurchase.getMoney(price)
-
-    if (lottoPrice.get() < LOTTO_PRICE) {
-        outputView.cannotPurchaseLotto()
-        return
-    }
+    val lottoPrice = LottoPrice(price)
 
     // 구입금액에 따른 로또 개수 반환
-    val lottoCount = lottoPurchase.getLotto(lottoPrice)
+    val lottoCount = lottoPurchase.getLottoCount(lottoPrice)
     outputView.resultPurchaseLotto(lottoCount)
 
     // 로또 개수에 맞추어 로또 번호 반환
     val lottoTickets = lottoPurchase.getLottoTickets(lottoCount)
     outputView.resultLottoTickets(lottoTickets)
 
     // 지난주 로또 당첨번호 받기
-    val lastWinningNumber = inputView.printMsgAndReadValue("\n${Const.OutputMsg.LOTTO_NUM_MSG}")
+    val winningNumber = WinningNumber()
```

Comment:
지난주 로또 당첨번호와 보너스 번호를 가진 객체로 보는건 어떤가요. `WinningNumber` 객체가 좀 더 객체지향적으로 바뀔 것 같아요.
```suggestion
    val winningNumber = WinningNumber(lastWinningNumbers, bonusNumber)
```

객체지향적이라는 의미는 아래 내용을 참고해 보세요.

> 객체는 상태와 행동 그리고 식별자를 갖는다 (객체지향 사실과 오해)

https://yangbongsoo.gitbook.io/study/oo_fact_and_misunderstanding